orca/split.py

```diff
@@ -0,0 +1,28 @@
+import re
+
+
+def split_url(url, size):
+    start_end_format = re.compile(r"(([a-z]+)\[(\d*)(:\d*){0,1}:(\d*)\])")
+    var_time, var, start, stride, end = start_end_format.findall(url)[0]
```

Comment:
Does this still work if there is not a stride?

Reply:
ya stride is just `''` in that case, and still works with the f-string I'm using